tests/tape_tests/conftest.py

```diff
@@ -2,14 +2,198 @@
 import numpy as np
 import pandas as pd
 import dask.dataframe as dd
-
 import pytest
+import tape
+
 from dask.distributed import Client
 
 from tape import Ensemble
 from tape.utils import ColumnMapper
 
 
+def create_test_rows():
```

Comment:
You could also make *this* a fixture, and create other fixtures with this fixture as input. Something like:

```
@pytest.fixture
def test_source_rows():
  ...

@pytest.fixture
def test_source_table(test_source_rows):
  ...

@pytest.fixture
def read_dask_dataframe_ensemble(dask_client: Client, test_source_table):
    return tape.read_dask_dataframe(
        dask_client = dask_client,
        source_frame = test_source_table,
        ...
    )
```

Reply:
Thanks, that's what I was originally going for but wasn't sure how to do it. I've made that change now so that those helper functions are now a @pytest.fixture.